protobuf/datastore.proto

```diff
@@ -123,7 +124,6 @@ message ObsMetadata {
   string id = 6;
   string data_id = 7 [json_name = "data_id"];
   string history = 8;
-  string metadata_id = 9 [json_name = "metadata_id"];
   string processing_level = 10 [json_name = "processing_level"];
```

Comment:
Should this be 9 now?

Reply:
No, field numbers only have to be unique within the message (see [this guide](https://protobuf.dev/programming-guides/proto3/)). Note also that as long as it is easy to rebuild all components that depend on datastore.proto (i.e. currently the ingestor and EDR API), we can freely alter the field numbers.

Reply:
Good :)